bundles/org.openhab.binding.solax/README.md

```diff
@@ -98,13 +98,13 @@ If you're missing a channel this means that it's not supported for your inverter
 | total-battery-discharge-energy   | Number:Energy              | Total energy from the battery [kWh]                       |
 | total-battery-charge-energy      | Number:Energy              | Total energy to the battery [kWh]                         |
 | total-pv-energy                  | Number:Energy              | Total energy from the PV [kWh]                            |
-| total-consumption                | Number:Energy              | Total energy consumed for the building [kWh]              |
-| total-feed-in-energy             | Number:Energy              | Total energy consumed from the electricity provider [kWh] |
+| total-consumption                | Number:Energy              | Total energy consumed by the building [kWh]               |
+| total-feed-in-energy             | Number:Energy              | Total energy sent to the grid [kWh]                       |
```

Comment:
I would expect `in` to correlate with Energy from the grid to the house.

In this case you are sure `in` means the opposite direction , from the house to the grid?!

Reply:
Yes, I'm sure. My readings show it. The usual nomenclature is: A PV-plant feeds its energy into the grid, a boiler or toaster consumes it from the grid.